src/JSInterop/Microsoft.JSInterop/src/JSRuntimeExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static async ValueTask InvokeVoidAsync(this IJSRuntime jsRuntime, string
                 throw new ArgumentNullException(nameof(jsRuntime));
             }
 
-            await jsRuntime.InvokeAsync<object>(identifier, args);
+            await jsRuntime.InvokeVoidAsync(identifier, args);
```

Comment:
These methods are currently calling themselves which results in infinite recursion.
I think this also needs to be exposed on the `IJSRuntime` interface or use `VoidReturn` here and make it internal / public.

Reply:
Doh!  You're right.  I thought about adding it to the interface as well but didn't since the current implementation just used extension methods.  I think it makes sense to add it to the interface at this point... @SteveSandersonMS thoughts?

Reply:
Hearing no response I went ahead and added the methods to the interface. @SteveSandersonMS is there anything else you are looking for/need for this PR to proceed?

Reply:
Thanks for following up on this. I know this PR has been delayed for a long time due to slow reviews on our side, so I'd be keen to work with you and get this merged in the next few days or next week, depending on what works best for you.

Apart from my suggestion above about avoiding the breaking change, the main remaining thing would be adding a few suitable test cases into the `test` project. Is that something you feel able to handle?